EzMap.Domain/Migrations/20231107153207_PoiItinerary.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace EzMap.Domain.Migrations
+{
+    /// <inheritdoc />
+    public partial class PoiItinerary : Migration
```

Comment:
Like we discussed, a List is just a List.
For instance you want to save a list of favorite restaurants, is that itinerary?
In its purest form, it's just a collection, for what purpose is it used depends on the user.
We will support "Itinerary" type of list (and many more), but then again, that will be just 
a "ListType" or "CollectionType".